doc/rtd/reference/network-config-format-v2.rst

```diff
@@ -309,69 +309,69 @@ Example: ::
       interfaces: [switchports]
 
 ``parameters: <(mapping)>``
-^^^^^^^^^^^^^^^^^^^^^^^^^^^
+---------------------------
 
 Customisation parameters for special bonding options. Time values are
 specified in seconds unless otherwise specified.
 
 ``mode: <(scalar)>``
-^^^^^^^^^^^^^^^^^^^^
+--------------------
```

Comment:
`mode` through `learn-packet-interval` depth should be unchanged as they all live under parameters.